Stripe/STPAddress.m

```diff
@@ -347,6 +354,48 @@ + (PKAddressField)applePayAddressFieldsFromBillingAddressFields:(STPBillingAddre
     }
 }
 
++ (PKAddressField)pkAddressFieldsFromStripeContactFields:(NSSet<STPContactField> *)contactFields {
+    PKAddressField addressFields = PKAddressFieldNone;
+    NSDictionary<STPContactField, NSNumber *> *contactToAddressFieldMap
+    = @{
+        STPContactFieldPostalAddress: @(PKAddressFieldPostalAddress),
+        STPContactFieldEmailAddress: @(PKAddressFieldEmail),
+        STPContactFieldPhoneNumber: @(PKAddressFieldPhone),
+        STPContactFieldName: @(PKAddressFieldName),
+        };
+
+    for (STPContactField contactField in contactFields) {
+        NSNumber *boxedConvertedField = contactToAddressFieldMap[contactField];
+        if (boxedConvertedField) {
+            addressFields = (PKAddressField) (addressFields | [boxedConvertedField unsignedIntegerValue]);
+        }
+    }
+    return addressFields;
+}
+
++ (NSSet<PKContactField> *)pkContactFieldsFromStripeContactFields:(NSSet<STPContactField> *)contactFields API_AVAILABLE(ios(11.0)) {
```

Comment:
same as above, `API_AVAILABLE(ios(11))`

Reply:
Hmm, the documentation for `API_AVAILABLE` seems to say you should do it as `11.0` and not `11`. Maybe a bug in faux pas? I also notice we do `NS_AVAILABLE_IOS` everywhere else, so maybe I'll just switch to that.